spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -6,7 +6,9 @@ def verify_identity_with_doc_auth
     click_on 'Create an account'
     create_new_account_with_sms
     expect(page).to have_current_path('/verify/doc_auth/welcome')
+    click_on 'Continue'
 
+    expect(page).to have_current_path('/verify/doc_auth/agreement')
```

Comment:
🙏 so glad we have the smoketests running on locally now, this would take so much longer to debug

Reply:
yes...thanks to YOU! :-)